test/client-old/spec/controllers/userCtrlSpec.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+describe.only('User Controller', function() {
+  var $rootScope, shared, scope, user, User, ctrl, content, achievement;
+
+  beforeEach(function() {
+    user = specHelper.newUser({
+      balance: 4,
+      items: {
+        gear: { owned: {} },
+        eggs: { Cactus: 1 },
+        hatchingPotions: { Base: 1 },
+        food: { Meat: 1 },
+        pets: {},
+        mounts: {}
+      },
+      flags: {
+        armoireEnabled: true
+      },
+      preferences: {
+        suppressModals: {}
+      },
+      purchased: {
+        plan: {
+          mysteryItems: [],
+        },
+      },
+    });
+
+    User = {
```

Comment:
You will want to provide this: https://github.com/HabitRPG/habitica/blob/develop/test/client-old/spec/controllers/authCtrlSpec.js#L7